CODE_OF_CONDUCT.md

```diff
@@ -118,14 +118,14 @@ version 2.1, available at
 [https://www.contributor-covenant.org/version/2/1/code_of_conduct.html][v2.1].
 
 Community Impact Guidelines were inspired by
-[Mozilla's code of conduct enforcement ladder][Mozilla CoC].
+[Mozilla's code of conduct enforcement ladder][mozilla coc].
```

Comment:
I would leave it as it is here

Reply:
Ok, thank you for highlighting this, my Prettier automatically formatted this. I will fix it and push again. I am contributing for the first time and want to know if my Pull request was done the right way.